packages/shared/src/components/RichTextEditor/plugins/shared/LinkSelector/PageLink.tsx

```diff
@@ -0,0 +1,83 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { AppBridgeBlock, useDocumentSection } from '@frontify/app-bridge';
+import { merge } from '@frontify/fondue';
+import React, { useEffect, useState } from 'react';
+import { InitiallyExpandedItems } from '../../LinkPlugin/FloatingLink/InsertLinkModal/types';
+import { SectionLink } from './SectionLink';
+
+type DocumentLinkProps = {
+    page: {
+        id: number;
+        title: string;
+        permanentLink: string;
+    };
+    selectedUrl: string;
+    onSelectUrl: (url: string) => void;
+    appBridge: AppBridgeBlock;
+    itemsToExpandInitially: InitiallyExpandedItems;
+};
+
+export const PageLink = ({ page, selectedUrl, onSelectUrl, itemsToExpandInitially, appBridge }: DocumentLinkProps) => {
```

Comment:
same as in `DocumentLink`.

We could even use a separat Component for the first part before `{isExpanded && (` and reuse in DocumentLink, PageLink & SectionLink. Wdyt?

Reply:
I like to keep them separate. Seems simpler to me, as all of them also have slightly different styling.